### DISCUSSION TITLE
Discuss adding Artic Wolf NX-OS schema to OVAL 6, or merging all similar Cisco schema's into a single schema

### DISCUSSION THREAD

**Discussion by vanderpol** [OVAL-Community/OVAL#208] [General]
During the discussion #165 it was mentioned by @maxullman  that the following tests were being used:

nxos line_test
nxos section_test

These were part of Artic Wolf developed NX-OS schema.  I mentioned that the vast majority of the Cisco IOS/IOSXE/NX schemas appear to be highly redundant, and we should consider if in OVAL 6.0 we should merge them all together.  @solind mentioned that any changes such as this should be for OVAL 6.0

Looking for opinions here:

1.  Add NX-OS schema to 6.0
2. Merge all Cisco schema's
3. Do nothing

**Comment by solind** [comment]
My vote would be for option 1.  Merging the schemas would be nice, but it would also require someone to do the work of deciding how to best accomplish that merge, and I don't think anyone wants to do that work.

**Reply by vanderpol** [reply]
I'll second that, although once I get done taking an axe to all of the unused tests, maybe merging the last X cisco items won't be that bad we'll see.  For now I'll plan for option 1
